citi-onboarding/acervusej-onepage
# feat/aboutus component

**Things I've done:**

- Create a responsive about us section
- I got the about us data from the database via backend route and used on the front end.

**How to test it:**

- First download the node modules on the client and the server. If you have already done that, then:
- Change the values on .env.
- Go to client on the terminal (cd client) and put npm run start.
- On the browser write http://localhost:3000.
- See if the about us section works both in mobile and desktop view.
- Look at the code to see if it has everything that it should have.
